packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -2229,7 +2224,24 @@ export class GLTFLoader implements IGLTFLoader {
     /**
      * @internal
      */
-    public _loadMaterialAsync(
+    private async _ensurePbrMaterialClassesAsync(): Promise<void> {

```

Comment:
We have a `Lazy` helper class now (added recently) that can be used for this type of thing. The Lazy instance can be created at loader construction time and then just accessed in `_loadMaterialAsync`. Up to you, but I suspect it would be a bit cleaner.

Reply:
Done